examples/nlp_seq/README.md

```diff
@@ -21,7 +21,7 @@ The model should run with other configurations and hardware, but explicitly test
 
 | Hardware |  Batch size  | Learning rate | Training time | Accuracy  | TensorBoard.dev |
 |:---:|:---:|:---:|:---:|:---:|:---:|
-| Nvidia V100 (16GB) | 64  |  0.05 | 5h 15m | 72.20% | [2020-03-22](https://tensorboard.dev/experiment/YkUAdwYaQ9OtYl2IVe3MvA/) |
+| Nvidia Titan V (12GB) | 64  |  0.05 | 41m | 92.5% | [2022-02-18](https://tensorboard.dev/experiment/dZTE3FoYSxarf29dKji10w/) |
```

Comment:
The traces look markedly different:
1. `accuracy/xpos_train` was ~1.00 previously, now is ~0.1
2. `accuracy/xpos_eval` was ~0.7 previously, now is ~0.9
3. `best_dev_score` was ~0.72 previously, now is ~0.92
4. `accuracy` measurements have much less variance
5. `best_dev_score` was fewer "jumps" than before

What happened here?

Reply:
Will be examined separately in #2201